.evergreen/generate_evergreen_tasks.js

```diff
@@ -101,11 +101,21 @@ MONGODB_VERSIONS.forEach(mongoVersion => {
 });
 
 // singleton task for connectivity tests
-TASKS.push({
-  name: 'test-atlas-connectivity',
-  tags: ['atlas-connect'],
-  commands: [{ func: 'install dependencies' }, { func: 'run atlas tests' }]
-});
+TASKS.push(
```

Comment:
can't push multiple items we use some weird prototype concat somewhere in here for this.

Reply:
nvm